src/current/cockroachcloud/managed-backups-advanced.md

```diff
@@ -29,10 +29,11 @@ Configurable managed backup settings are available in all [supported versions]({
 
 Full backups in the cluster will be deleted when they reach the set retention period. At this point, any incremental backups dependent on the deleted full backup will also be deleted. The Cloud Console will not list any backups that are beyond the set retention period, or incremental backups that cannot be restored.
 
-For instructions on how to view and configure managed backup settings, use:
+For instructions on how to view and configure managed backup settings, use the:
```

Comment:
do we want folks to use "one of" the following? or is it a mix?  if the former i'd suggest saying "use one of the following"

Reply:
Changed!